modules/azure/datafactory.go

```diff
@@ -0,0 +1,36 @@
+package azure
+
+import (
+	"context"
+
+	"github.com/Azure/azure-sdk-for-go/services/datafactory/mgmt/2018-06-01/datafactory"
+	"github.com/gruntwork-io/terratest/modules/testing"
+	"github.com/stretchr/testify/require"
+)
+
```

Comment:
could you add a helper to check dataFactoryExists. Similar to here https://github.com/gruntwork-io/terratest/blob/60601b46a2e525968e64745443698abbae20daf5/modules/azure/frontdoor.go#L13

Reply:
Thanks @HadwaAbdelhalem added